app/services/symphony_client.rb

```diff
@@ -97,14 +97,14 @@ def renew_item(resource, item_key)
   end
 
   def renew_items(checkouts)
-    checkouts.each_with_object(success: [], error: []) do |checkout, flash|
+    checkouts.each_with_object(success: [], error: []) do |checkout, status|
       response = renew_item(checkout.resource, checkout.item_key)
 
       case response.status
       when 200
-        flash[:success] << I18n.t('mylibrary.renew_item.success_html', title: checkout.title)
+        status[:success] << checkout.title
```

Comment:
I'm 👍 with what's here, although I wonder about returning the whole checkout and letting the thing that receives it pluck out the title?

Reply:
👍 that makes sense.